channels/whatsapp/index.rst

```diff
@@ -0,0 +1,247 @@
+WhatsApp
+========
+
+.. toctree::
+   :hidden:
+
+   preparation
+
+With the release of Zammad 6.3, you can now connect your Zammad with a
+WhatsApp business account, allowing your customers to reach out to you through
+WhatsApp.  🎉
+
+If you already dived deep into this topic, you can head to the setup guide.
+If not, we strongly encourage you to read the prerequisites and limitations
+of this channel carefully and to follow our tutorial for preparing the
+required steps.
+
+We tried to keep it clear and created a separate sub-page for the necessary
+steps on Meta's/WhatsApp's side.
+
+.. warning::
+   Please be aware that the usage of **WhatsApp's cloud API** is mandatory to
```

Comment:
Again, the name is `WhatsApp Cloud API` (watch for capitalization), avoid using possessive forms like `*'s` here.